src/components/KPop/KPop.vue

```diff
@@ -116,18 +81,20 @@
 // @ts-nocheck
 import type { PropType } from 'vue'
 import { defineComponent } from 'vue'
-import { v1 as uuidv1 } from 'uuid'
+import { v4 as uuidv4 } from 'uuid'
 import Popper from 'popper.js'
 import useUtilities from '@/composables/useUtilities'
 import KButton from '@/components/KButton/KButton.vue'
 import type { PopPlacements, PopTrigger } from '@/types'
 import { PopPlacementsArray, PopTriggerArray } from '@/types'
+import { CloseIcon } from '@kong/icons'
+import { KUI_ICON_SIZE_30 } from '@kong/design-tokens'
 
 const { getSizeFromString } = useUtilities()
 
 export default defineComponent({
   name: 'KPop',
-  components: { KButton },
+  components: { KButton, CloseIcon },
```

Comment:
At first I was going to comment “why” but remembered we said we’d wait to switch to script setup when we refactor

Reply:
Yeah, there's a comment at the top of `script` section saying that